README.md

```diff
@@ -1,3470 +1,188 @@
-
-The `evictionMaxSize` defines the entry count when the Near Cache is full and determines whether the eviction should be triggered.
-
-Once the eviction is triggered the configured `evictionPolicy` determines which, if any, entries must be evicted.
-
-#### 7.8.2.4. Near Cache Expiration
-
-Expiration means the eviction of expired records. A record is expired:
-
-- if it is not touched (accessed/read) for `maxIdleSeconds`
-
-- `timeToLiveSeconds` passed since it is put to Near Cache
-
-The actual expiration is performed when a record is accessed: it is checked if the record is expired or not. If it is expired, it is evicted and `undefined` is returned as the value to the caller.
-
-
-#### 7.8.2.5. Near Cache Invalidation
-
-Invalidation is the process of removing an entry from the Near Cache when its value is updated or it is removed from the original map (to prevent stale reads). See the [Near Cache Invalidation section](https://docs.hazelcast.org/docs/latest/manual/html-single/#near-cache-invalidation) in the Hazelcast IMDG Reference Manual.
-
-#### 7.8.2.6. Near Cache Eventual Consistency
-
-Near Caches are invalidated by invalidation events. Invalidation events can be lost due to the fire-and-forget fashion of eventing system. If an event is lost, reads from Near Cache can indefinitely be stale.
-
-To solve this problem, Hazelcast provides eventually consistent behavior for Map Near Caches by detecting invalidation losses. After detection of an invalidation loss, stale data will be made unreachable and Near Cache’s `get` calls to that data will be directed to underlying Map to fetch the fresh data.
-
-You can configure eventual consistency with the `ClientConfig.properties` below:
-
-- `hazelcast.invalidation.max.tolerated.miss.count`: Default value is `10`. If missed invalidation count is bigger than this value, relevant cached data will be made unreachable.
-
-- `hazelcast.invalidation.reconciliation.interval.seconds`: Default value is `60` seconds. This is a periodic task that scans cluster members periodically to compare generated invalidation events with the received ones from the client Near Cache.
-
-### 7.8.3. Automated Pipelining
+## Overview
 
-Hazelcast Node.js client performs automated pipelining of operations. It means that the library pushes all operations into an internal queue and tries to send them in batches. This reduces the count of executed `Socket.write()` calls and significantly improves throughtput for read operations.
+### Usage
 
-You can configure automated operation pipelining with the `ClientConfig.properties` below:
+```js
+const Client = require('hazelcast-client').Client;
 
-- `hazelcast.client.autopipelining.enabled`: Default value is `true`. Turns automated pipelining feature on/off. If your application does only writes operations, like `IMap.set()`, you can try disabling automated pipelining to get a slightly better throughtput.
+// Connect to Hazelcast cluster
+Client.newHazelcastClient()
```

Comment:
It's better to use async/await syntax here for the sake of readability.

The same suggestion applies to the config snippet.

Reply:
Should we put the code samples into an async function or should they be in the top-level for demonstration purposes ?

Reply:
I'd prefer top-level as it'll make snippets more readable. Any reader who is familiar with async/await knows that they need to wrap the snippet into an async function.